### PR TITLE
Update DG

### LINKED ISSUE #132: Last fixes for DG
Need to add FindOurSEP main class component in DG, fix the printStudentList sd (Activation bars required), add find and filter command class diagrams, and have an ice-cold cat picture in about us.

### PR DESCRIPTION
**Pull request by Holy-An**
close #132 

- Added FindOurSEP main class component in DG
- Fixed the printStudentList sd (Activation bars required)
- Added find and filter command class diagrams
- Updated about us.

### CODE REVIEW

File: docs/UML_Diagrams/saveAllocationResultsSequence.drawio.svg
Comment (ehz0ah) [inline docs/UML_Diagrams/saveAllocationResultsSequence.drawio.svg:1]: Should there be a base activation bar for the filehandler class and separate ones that overlay the base for self called methods?

File: docs/images/Catnaldo.jpeg
Comment (ehz0ah) [inline docs/images/Catnaldo.jpeg:1]: cool

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -292,10 +300,33 @@ Note: For further details on the `FileHandler` class / `setUpFileHandler()` meth
 
 
 ## Components
+FindOurSEP is primarily a Command-Line Interface (CLI) based Java Application. The entry point to our application is
```
Comment (ehz0ah) [inline docs/DeveloperGuide.md:303]: Should all of this just move to the top of the DG instead?

File: META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Manifest-Version: 1.0
```
Comment (ehz0ah) [inline META-INF/MANIFEST.MF:1]: What is this new file?

File: docs/UML_Diagrams/saveAllocationResultsSequence.drawio.svg
Comment (Holy-An) [reply docs/UML_Diagrams/saveAllocationResultsSequence.drawio.svg:1]: It's a reference into saveAllocationResults() so I think it should be fine

File: META-INF/MANIFEST.MF (hunk 2 above)
Comment (Holy-An) [reply META-INF/MANIFEST.MF:1]: This was generated from the making of the sep jar file 😅, should have git ignored it mb

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Holy-An) [reply docs/DeveloperGuide.md:303]: That works as well

**Review by ehz0ah — APPROVED** [review]
ACK

**Pull request merged**